shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -89,12 +124,12 @@ public String getDescription() {
         + "Where category is an optional argument, if no arguments passed in, "
```

Comment:
no arguments passed in -> no arguments are passed in

Reply:
fixed, thanks